index.html

```diff
@@ -1,8 +1,27 @@
 <html>
     <head>
-      <script src="index.js"></script>
+      <script src="index.js" defer></script>
     </head>
     <body>
-      <h1> CODE YOUR SOLUTION </h1>      
+      <h1>My To-Dos</h1>    
+      <hr>  
+    <br/>
+    <ul>
+
+    </ul>
+    <form>
+      <label
+      >To-Do:
+      <input
+      id="todo"
+      name="todo"
+      type="text"
+      placeholder="Add an item to list."
+      required
+      />
+      <br/>
+      <br/>
+  <input type="submit" value="Add to List">
```

Comment:
In order to pass the specs, you need a button of type to submit and not an input, but it matters only for the way we wrote the tests and not for the actual functionality.